main/src/modules/reactiveLayer/sensationManager/include/opcSensation.h

```diff
@@ -29,11 +29,15 @@ class OpcSensation: public Sensation
     void addToEntityList(yarp::os::Bottle& list, std::string type, std::string name);
     Bottle handleEntities();
     void handleTouch();
+    
 
 public:
 
+    Bottle u_entities, k_entities, up_entities, kp_entities, p_entities;

```

Comment:
Any reason why these variables are global? I think they should be local instead.